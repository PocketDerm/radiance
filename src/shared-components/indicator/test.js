import React from 'react';
import renderer from 'react-test-renderer';

import Indicator from './index';

describe('<Indicator />', () => {
  describe('UI snapshots', () => {
    it('renders the correct css and text', () => {
      const component = renderer.create(<Indicator text="1" />);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
