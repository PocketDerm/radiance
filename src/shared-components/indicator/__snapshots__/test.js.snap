// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Indicator /> UI snapshots renders the correct css and text 1`] = `
.emotion-0 {
  background-color: #BD200F;
  min-height: 16px;
  height: 16px;
  max-height: 16px;
  text-align: center;
  padding: 0 0.25rem;
  min-width: 16px;
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 8px;
}

.emotion-0 > div {
  position: relative;
  top: 1px;
  color: #524D6E;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  color: #ffffff;
}

<div
  className="emotion-0 emotion-1"
>
  <div>
    1
  </div>
</div>
`;
